elastic/kibana
# [Serverless] make parent nav item into breadcrumb link

## Summary

Currently the breadcrumb of the parent navigation item isn't a link:

<img width="339" alt="Screenshot 2024-05-10 at 2 06 42 PM" src="https://github.com/elastic/kibana/assets/1676003/1471125f-59d7-468e-8ede-3a3c3c3e42dc">

This change makes it so the breadcrumb of parent navigation item is linkable:
<img width="293" alt="Screenshot 2024-05-10 at 2 21 27 PM" src="https://github.com/elastic/kibana/assets/1676003/aad5e650-33e1-419d-b866-0b7ec70e1f42">

The links will go to the first item in the list of child links.

The merging of this PR depends on https://github.com/elastic/kibana/pull/183508 allows for this functionality to work without making the left side parent collapsible link, also a href link - only the breadcrumb.

## Testing
- Start a serverless kibana instance

- Navigate to a child link of an accordion using the left side navigation
<img width="589" alt="Screenshot 2024-05-15 at 11 26 24 AM" src="https://github.com/elastic/kibana/assets/1676003/d225967d-37d1-4f19-b3f5-856fcc1bce68">

- The breadcrumb should of this page should be a link and link you to the first child link of the accordion
